src/JsonValidator.php

```diff
@@ -135,4 +152,14 @@ private function loadSchema($file)
 
         return $schema;
     }
+
+    /**
+     * @return Validator
+     */
+    private function prepareValidator()
```

Comment:
Thanks for updating! :) I don't think this needs to live in a separate method. You can simply inline the `reset()` call above.